sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -14,6 +14,7 @@ import { MessagingError } from '@azure/core-amqp';
 import { OperationOptions } from '@azure/core-auth';
 import { RetryOptions } from '@azure/core-amqp';
 import { TokenCredential } from '@azure/core-amqp';
+import { TokenCredential as TokenCredential_2 } from '@azure/core-auth';
```

Comment:
Are they both required?

Reply:
Ha, I'm guessing 'no'.

I looked at EH and it looks like we're just getting it straight from core-amqp. Swapped.